src/components/Footer.jsx

```diff
@@ -1,48 +1,130 @@
+/* eslint-disable no-console */
 import { Input, Button, Textarea } from '@nextui-org/react';
-import { FaSquareInstagram, FaSquareFacebook } from 'react-icons/fa6';
+import axios from 'axios';
+import { useState } from 'react';
+import toast from 'react-hot-toast';
+import { FaSquareInstagram, FaSquareFacebook, FaSquarePhone, FaLocationDot } from 'react-icons/fa6';
 import { FiMail, FiUser } from 'react-icons/fi';
+import { MdEmail } from 'react-icons/md';
 
+// FOOTER COMPONENET
 export default function Footer() {
+  // Set state for form data
+  const [formData, setFormData] = useState({
+    name: '',
+    email: '',
+    questions: ''
+  });
+
+  // Handle changes in input fields
+  const handleInputChange = (e) => {
```

Comment:
Why don't you use `react-hook-form` here and add some validation?

Reply:
honestly, I don't think this is necessary for data validation since it does not go to database and not some critical data. Sorry I didn't really think of react-hook-form since I just wanted to do a quick one, make it work and move on.

Reply:
I think it's better to make thing align across the app as we already been using `react-hook-form` for all the form and this should be easy to implement too. If you don't wanna use it just move on